programs/axelar-solana-its/src/instruction.rs

```diff
@@ -1880,10 +1881,12 @@ fn derive_specific_its_accounts(
                     &mint_account,
                     &token_program,
                 );
+                let interchain_transfer_execute_key = find_interchain_transfer_execute_pda(&find_its_root_pda().0, &destination_account).0;
 
                 specific_accounts.push(AccountMeta::new(program_ata, false));
                 specific_accounts.push(AccountMeta::new_readonly(mpl_token_metadata::ID, false));
                 specific_accounts.push(AccountMeta::new(metadata_account_key, false));
+                specific_accounts.push(AccountMeta::new(interchain_transfer_execute_key, false));
```

Comment:
```suggestion
                specific_accounts.push(AccountMeta::new(interchain_transfer_execute_key, true));
```
I believe this should be signer, otherwise anyone could just reuse this account after it's initialized

Reply:
nvm, ITS will sign it on-chain, the relayer cannot sign on its behalf